analysis/druidbalance/src/modules/Abilities.tsx

```diff
@@ -165,6 +160,22 @@ class Abilities extends CoreAbilities {
         },
         timelineSortIndex: 11,
       },
+      {
+        spell: SPELLS.CONVOKE_SPIRITS,
+        buffSpellId: SPELLS.CONVOKE_SPIRITS.id,
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        cooldown: 120,
+        enabled: combatant.hasCovenant(COVENANTS.NIGHT_FAE.id),
+        gcd: {
+          base: 1500,
+        },
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.85,
+          majorIssueEfficiency: 0.80
+        },
+        timelineSortIndex: 12,
+      },
```

Comment:
I'm wondering if this will interfere with this block here in the convoke module: https://github.com/WoWAnalyzer/WoWAnalyzer/blob/05bef0cec0d64bd8a9c355dfb02061935b87d368/analysis/druid/src/ConvokeSpirits.tsx#L58-L69

Reply:
Thanks for the reminder, it does indeed lead to the following:
![image](https://user-images.githubusercontent.com/36763177/107874829-ab31b280-6ebc-11eb-9f92-e4e792386c67.png)

While I am not quite sure if the above way of injecting it into the options.abilities might lead to problems later on (Spec specific requirements on efficiency for example), I can resolve it for the time being by removing my convoke definition on the Abilities.tsx and inserting the ConvokeSpirits.tsx statistic (which should be done later on either way).

Reply:
Injecting it into the `options.abilities` is fine, it doesn't really make much of a difference. Injecting it like that is generally used in cases where a deeper analysis is required on the `maxCasts` and you need more calculation to go into it (ie: murder of crows resets on target's death, void bolt only available in voidform).